pages/managed-databases-for-postgresql-and-mysql/api-cli/logical-replication-as-subscriber.mdx

```diff
@@ -0,0 +1,105 @@
+
+    ```sql
+    GRANT pg_create_subscription TO my_replication_user;
+    GRANT CREATE ON DATABASE my_database TO my_replication_user;
+    ```
+2. Create a publication in the original database to define which tables and schemas will be replicated. In the example below we include all tables in the database.
+    <Message type="tip">
+      Refer to the official [Create Publication](https://www.postgresql.org/docs/16/sql-createpublication.html) PostgreSQL documentation to learn how to detail the specific tables or schemas.
+    </Message>
+
+    ```sql
+    CREATE PUBLICATION my_publication FOR ALL TABLES;
+    ```
+3. Subscribe the subscriber database to the publication created in the previous step. This will create a subscription that connects both databases.
+    <Message type="important">
+      Subscriptions in the same database cannot have the same name.
+    </Message>
+
+    ```sql
+    CREATE SUBSCRIPTION my_subscription
+      CONNECTION 'host=publisher_host port=5432 user=my_replication_user password=my_password dbname=my_database'
+      PUBLICATION my_publication;
+    ```
+4. Run the commands below to respectively grant the user read access to the subscription, and to check its statistics. This helps you make sure the subscription was successful. You can check the number of pending changes and last error message in the output.
+    ```sql
+    GRANT pg_read_all_stats TO my_replication_user;
+    SELECT * FROM pg_stat_subscription;
+    ```
+5. Run the first command to grant the user read access to all data. Run the second to get a table containing data about all subscriptions defined in the database. This data includes the subscription ID, name, owner, whether the subscription is enabled or not, and connection information.
+    ```sql
+    GRANT pg_read_all_data TO myuser;
+    SELECT * FROM pg_subscription;
+    ```
+    <Message type="tip">
+      Refer to the official PostgreSQL documentation for more information on the `pg_subscription` catalog.
+    </Message>
+
+
+## Limitations
+
+When creating a subscription from a Scaleway Database Instance to a public IP, the public interface of the Database Instance node is used to connect to the publisher. This might incur in the following consequences:
+
+- If you only have a private endpoint, your Database Instance will still has a public management IP address. The ports are all closed, stopping all connections except the one with the publisher or subscriber database.
+- If your Database Instance set-up is composed of more than one node, your IP address is subject to change. In case of failover, for example, the IP address connecting to the publisher database will change. We cannot predict the new IP your nodes might take one. However, you can use [Scaleway's Autonomous System](https://ipinfo.io/AS12876) to check the number (ASN) of your primary node's IP address, look up the IP address allocation for the ASN. This can help you predict the IP address of your failover nodes.
```

Comment:
```suggestion
- If your Database Instance set-up is composed of more than one node, your IP address is subject to change. In case of failover, for example, the IP address connecting to the publisher database will change. We cannot predict the new IP your nodes might take. However, you can use [Scaleway's Autonomous System](https://ipinfo.io/AS12876) to check the number (ASN) of your primary node's IP address, and look up the IP address allocation for the ASN. This can help you predict the IP address of your failover nodes.
```